mono/metadata/locales.c

```diff
@@ -294,20 +300,20 @@ construct_culture (MonoCultureInfo *this_obj, const CultureInfoEntry *ci)
 }
 
 static MonoBoolean
-construct_region (MonoRegionInfo *this_obj, const RegionInfoEntry *ri)
+construct_region (MonoRegionInfoHandle* this_obj, const RegionInfoEntry *ri)
```

Comment:
Hmm, shouldn't we be passing handles by value? It's odd to pass them by reference.